src/common/low_precision_transformations/src/qdq_stripping.cpp

```diff
@@ -37,6 +37,7 @@ FQStrippingTransformation::FQStrippingTransformation(const std::set<size_t>& lev
         {pattern::any_input(), input_low_m, input_high_m, output_low_m, output_high_m});
 
     ov::graph_rewrite_callback callback = [OV_CAPTURE_CPY_AND_THIS](pattern::Matcher& m) {
+        (void)this;
```

Comment:
It should be removed, don't capture this

Reply:
That creates a warning about deprecated `[=]` capture group usage. Generally, I agree, but the compiler behavior is like that